packages/generator-electrode-component/app/index.js

```diff
@@ -136,6 +145,9 @@ var ReactComponentGenerator = yeoman.generators.Base.extend({
 	},
 
 	end: function() {
+		if (this.quoteType === "'") {
+			this.spawnCommandSync("node_modules/.bin/eslint", ["--fix", "src", "demo", "example", "test", "--ext", ".js,.jsx"]);
```

Comment:
yes it is in the node_modules because of the archtypes. so you have to run this step after the this.npmInstall().